memlimit/memlimit.go

```diff
@@ -113,3 +164,14 @@ func Limit(limit uint64) func() (uint64, error) {
 		return limit, nil
 	}
 }
+
+func prettyByteSize(b int64) string {
+	bf := float64(b)
+	for _, unit := range []string{"", "Ki", "Mi", "Gi", "Ti", "Pi", "Ei", "Zi"} {
+		if math.Abs(bf) < 1024.0 {
```

Comment:
Is `math.Abs` necessary? What is this for?

Reply:
@KimMachineGun, done! delete `math.Abs`.